src/biokbase/narrative/jobs/jobcomm.py

```diff
@@ -232,9 +239,8 @@ def start_job_status_loop(
                     "code": getattr(e, "code", -1),
                     "source": getattr(e, "source", "jobmanager"),
                     "name": getattr(e, "name", type(e).__name__),
-                    "service": "execution_engine2",
                 }
-                self.send_comm_message("job_init_err", error)
```

Comment:
`job_init_err` isn't in the docs. Changed it for consistency.

Reply:
Also, keeping this as is, without delegating error handling to the context manager, because this gets called from the FE and from `jc._modify_job_updates` so I want to keep behavior as is and not skip code afterwards or lost messages